packages/doxdox-core/src/utils.test.ts

```diff
@@ -10,8 +10,7 @@ import {
     isDirectory,
     isFile,
     parseConfigFromCLI,
-    parseIgnoreConfig,
-    sanitizePath,
+    parseIgnoreConfig
```

Comment:
Everything looks good! I also updated the JSDoc parser to use the `fileURLToPath` method as it also used the `sanitizePath`. That fixed the tests on Windows for that plugin, so thank you again for submitting this PR!

The only issue I found was a missing comma after this import.

```suggestion
    parseIgnoreConfig,
```